application/tests/features/api.roles.feature

```diff
@@ -94,3 +94,52 @@ Feature: Testing the Roles API
         And the "results.0.name" property equals "manager"
         Then the guzzle status code should be 200
 
+     Scenario: Get protected status of protected role
+        Given that I want to find a "Role"
+        And that the request "Authorization" header is "Bearer testadminuser"
+        And that its "id" is "1"
+        When I request "/roles"
+        Then the response is JSON
+		And the response has a "protected" property
```

Comment:
Possible. I use Coda cuz I like some of the tooling for code syntax, change management, search and ssh, but the auto indentation "feature" on ⏎ can be a bit buggy.